apps/src/lab2/projects/ProjectContainer.tsx

```diff
@@ -28,6 +28,9 @@ const ProjectContainer: React.FunctionComponent<ProjectContainerProps> = ({
   const currentLevelId = useSelector(
     (state: {progress: ProgressState}) => state.progress.currentLevelId
   );
+  const userId = useSelector(
+    (state: {progress: ProgressState}) => state.progress.userId || ''
+  );
```

Comment:
nit: can use `useAppSelector` here to skip the type cast (but also fine if we want to keep it consistent with the rest and update them all later)

Reply:
Good call, though in the spirit of reducing changes, might be good for a potential follow-up.